pages/community.tsx

```diff
@@ -0,0 +1,397 @@
+                backgroundClip="text"
+                color="transparent"
+              >
+                decentralized talent.
+              </Text>
+            </Heading>
+
+            <LinkButton
+              href="https://discord.gg/thirdweb"
+              size="lg"
+              onClick={() =>
+                trackEvent({
+                  category: TRACKING_CATEGORY,
+                  action: "click",
+                  title: "Join the community",
+                })
+              }
+              background="white"
+              _hover={{
+                background: "white",
+                color: "#000",
+              }}
+              color="#000"
+              fontSize="18px"
+              leftIcon={<Icon as={FiZap} />}
+              mt={8}
+            >
+              Join the community
+            </LinkButton>
+          </Center>
+
+          {CommunitySections.map(
+            ({ title1, title2, cards, description }, i) => (
+              <Box
+                w="100%"
+                as="section"
+                zIndex={2}
+                position="relative"
+                key={title1}
+              >
+                <Box
+                  h="1px"
+                  bg="#3F3F3F"
+                  opacity="0.8"
+                  maxW="container.page"
+                  display={i === 0 ? "none" : "block"}
+                  mx="auto"
+                />
+                <Container
+                  maxW="container.page"
+                  position="relative"
+                  py={{ base: 12, md: 24 }}
+                >
+                  <Heading
+                    as="h2"
+                    textAlign="left"
+                    fontSize="48px"
+                    color="#ECECEC"
+                  >
+                    {title1}{" "}
+                    <Text
+                      as="span"
+                      fontSize="48px"
+                      color="transparent"
+                      fontWeight="bold"
+                      background="linear-gradient(249.19deg, #BFA3DA 53.72%, #84309C 78.01%, #C735B0 112.02%), #ECECEC"
+                      backgroundClip="text"
+                    >
+                      {title2}
+                    </Text>
+                  </Heading>
+
+                  <Text
+                    mt={2}
+                    textAlign="left"
+                    fontSize="20px"
+                    color="white"
+                    opacity={0.7}
+                    maxW="container.md"
+                  >
+                    {description}
+                  </Text>
+
+                  <SimpleGrid
+                    columns={{ base: 1, md: 4 }}
+                    gap={{ base: 12, md: 6 }}
+                    py={{ base: 12, md: 8 }}
+                  >
+                    {cards.map(
+                      ({
+                        image,
+                        title: cTitle,
```

Comment:
```suggestion
                        title: cardTitle,
```
no need to short name that much